### PR TITLE
updated package.json (added description, author, repository and homepage)

### PR DESCRIPTION
**Pull request by boyuan12**
## Description
Updated package.json accordingly. I added Description, Author, Repository and Homepage to package.json
## Related issues and discussion
#3127

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

-  [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,11 @@
 {
   "name": "mifosx-community-app",
+  "description": "This is the default web application built on top of the MifosX platform for the mifos user community. It is a Single-Page App (SPA) written in web standard technologies like JavaScript, CSS and HTML5. It leverages common popular frameworks/libraries such as AngularJS, Bootstrap and Font Awesome.",
+  "author": {
+    "name": "openMF",
+    "email": "edcable@mifos.org",
```
Comment (OussEmaDevCode) [inline package.json]:
```suggestion 
    "email": "mifos-developer@lists.sourceforge.net",

Comment (OHassini) [inline package.json]:
```suggestion 
    "email": "mifos-developer@lists.sourceforge.net",

Comment (OHassini) [reply package.json]: That's edcable's email not the Mifos email.

**Comment by OHassini** [comment]
@boyuan12 Have a look at the review !!!

File: package.json (hunk 1 above)
Comment (boyuan12) [reply package.json]: Thank you, I actually don't know their email address so I used their CEO's email. Thanks for commenting!

**Comment by OHassini** [comment]
@boyuan12 Good job ! Now make sure you squash the commits (if you want help doing that you can ask).

**Comment by boyuan12** [comment]
Thank you, I do have some questions on how to squashing commits. According to this [gist](https://gist.github.com/laxyapahuja/e30921a9478bdf3a9e166a023ce1882c), So it said I should use `git rebase -i HEAD~2` to squashing last 2 commits. But it gives me the following errors: 
`error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.`
Could you help me, please? Thank you.

**Comment by OHassini** [comment]
@boyuan12 It seems like you have done some extra changes after pushing this commit. Try removing all the extra changes you have done and then run that command. If that does not work run 'git status' and share the result.

**Comment by boyuan12** [comment]
I'm still confused about how to squashing commit. Here is the result after running ```git status```
Screen Shot 2020-01-31 at 3 45 22 PM

**Comment by OHassini** [comment]
@boyuan12 Ok, try running this command `git reset --hard HEAD` to remove all extra changes then run the rebase command you mentioned above.

**Comment by boyuan12** [comment]
Could you check again? I used your command and then the rebase command, and I think it should work now. (Screenshot result attached)
Screen Shot 2020-01-31 at 11 16 15 PM
Thanks.

**Comment by OHassini** [comment]
@boyuan12 Good job, now force push your squashed commit via this command `git push -f`.

**Comment by boyuan12** [comment]
I did force commit, could you check again? Thanks.

**Comment by OHassini** [comment]
Somehow you have deleted the last commit instead of squashing it, in other words the requested changes have now been removed. To fix that, first of all change the email as I mentioned above then run `git add package.json`  then `git commit -m "fixC"` then `git rebase -i HEAD~2`, a text editor will open up ,there look for the " fixC" commit and replace "pick" with "squash" and save and quit. Once all of that is done run `git push -f` to force push.

**Comment by boyuan12** [comment]
Ok, so I followed your instructions and hopefully squash them. Please check again. thanks.

**Comment by OHassini** [comment]
You did not squash the commits (fixC commit is still visible), make sure you run this command `git rebase -i HEAD~2` before force pushing. (Send me a screenshot of the text editor if you're stuck)

**Comment by boyuan12** [comment]
Oh, I think I forgot to save "pick" to "squash" in my vi editor. Could you check again? I think it should work now. Thanks.

**Comment by OHassini** [comment]
@boyuan12 Good job !!! Let's see what @abhaychawla thinks...

**Comment by boyuan12** [comment]
@OussEmaDevCode Did you know usually when PR gets approved and merged?

**Pull request merged**